en/simulation/multi_vehicle_jmavsim.md

```diff
@@ -0,0 +1,52 @@
+# Multi-Vehicle Simulation with JMAVSim
+
+This topic explains how to simulate multiple UAV (multicopter) vehicles using JMAVSim and SITL.
```

Comment:
Should we mention that this runs multiple independent simulators and all vehicles start at the same position? Meaning if you want to do swarm simulation (or similar) you're better off using gazebo.

Reply:
Thanks @bkueng .

Can you clarify a bit more on why Gazebo is better for swarm? I ASSUME it is because 
* You can choose different vehicle start locations (?)
* The simulation runs a single instance of Gazebo with multiple SITL? (I presume you are saying that for JMAVSim `runs multiple independent simulators` means that you're running multiple SITL and multiple JMAVSim, and that therefore this is more increasingly resource consuming than Gazebo as you scale up.

Is that correct?

So presumably this is for when you have a small number of device - e.g. a 3-device survey and you don't care about the starting position - right?
At what number of instances would you guess that Gazebo becomes more reasonable?